index.html

```diff
@@ -210,22 +210,21 @@ <h3>New Technical Reports</h3>
     </dd>
   </dl>
 
-  <p>The Working Group MAY develop additional Recommendation-track and non-Recommendation-track Technical
-  Reports.</p>
+  <p>The Working Group MAY develop additional Recommendation-track and non-Recommendation-track specifications.</p>
 </div>
 
 <div id="revisions">
-  <h3>Existing Technical Reports</h3>
+  <h3>Existing Normative Specifications</h3>
 
   <p>The Working Group MAY update its <a href="https://www.w3.org/wiki/TimedText/Publications">previously
-  published Technical Reports</a>.</p>
+  published Normative Specifications</a>.</p>
```

Comment:
I'd revert this, because the WG may also update previously published Technical Reports that are not Normative Specifications.

Reply:
ah, how about to cover these by using `Other non-normative documents may be created such as: ` for our 2.4 other deliverables (section number before this PR merged)?

Reply:
Yes, if you change "Other non-normative documents may be created" to "Other non-normative documents may be created or updated" that would work.

Reply:
how about
https://github.com/himorin/charter-timed-text/blob/b81dcf89731d57e00ae83cb3f0c0d8a22fb494ed/index.html#L227

> <p>The Working Group MAY create documents that are not Normative Specifications, including:</p>

for the first line of `Other deliverables`?